luskan/src/main/java/org/opencadc/luskan/TempStorageManager.java

```diff
@@ -250,4 +263,45 @@ public Content accept(String name, String contentType, InputStream inputStream)
     private static String getRandomString() {
         return new RandomStringGenerator(16).getID();
     }
+
+    private void scheduleDeletion(final File file) {
+        getScheduler().schedule(new Runnable() {
+            @Override public void run() {
+                try {
+                    if (!file.exists()) {
+                        log.info(String.format("%s not found for deletion", file.getAbsolutePath()));
+                    } else {
+                        boolean deleted = file.delete();
+                        log.debug(String.format("deleted %s: %s", file.getAbsolutePath(), deleted));
+                    }
+                } catch (SecurityException e) {
+                    log.error(String.format("Unable to delete %s because %s", file.getAbsolutePath(),
+                                            e.getMessage()));
+                }
+            }
```

Comment:
The constants in QueryJobManager are in seconds.

Reply:
That would be about 10 minutes before deletion. Fixed.